src/ert/_c_wrappers/util/substitution_list.py

```diff
@@ -124,7 +124,9 @@ def substitute_real_iter(
             copy_substituter.addItem("<GEO_ID>", self[geo_id_key])
         copy_substituter.addItem("<IENS>", str(realization))
         copy_substituter.addItem("<ITER>", str(iteration))
-        return copy_substituter.substitute(to_substitute)
+        return copy_substituter.substitute(
+            to_substitute, "substituter with GEO_ID, IENS, and ITER"
```

Comment:
This context is probably more confusing than just keeping it empty.

Reply:
removed